src/firebase/index.js

```diff
@@ -0,0 +1,15 @@
+export const listenToMessages = (firestore) =>
+  firestore().collection("generalMessages").orderBy("createdAt");
+
+export const createMessage = (firestore, user, message) =>
+  firestore()
+    .collection("generalMessages")
+    .add({
+      message,
+      createdAt: new Date().getTime(),
```

Comment:
```suggestion
      createdAt: Date.now()
```
source: https://stackoverflow.com/a/12517433